### DISCUSSION TITLE
Can you change the print area bed size?

### DISCUSSION THREAD

**Discussion by Benvolio28** [Jyers/Marlin#451] [Q&A] [answered]
I just installed this firmware and while doing a mesh leveling for the first time my BL touch 3.1 was hanging off the side of the bed and was not touching, making the mesh leveling fail.  I noticed in the 'info' section the bed is listed as 230 x 230 x 250.  The Ender 3 V2 has a bed of 220 x 220 x 250, perhaps this is where the error lies as it was not too far off the bed.

I searched and did not find an answer.

Thanks for any help you can provide!

**Comment by tome9111991** [comment]
You can change bed size in configuration.h

**Comment by aryajuanda** [comment, marked as answer]
you need to set probe to nozzle offset so probe stay in bed area.. 230x230x250  + 15mm on the max X is better than default 220x220x250 :
1. you can have larger printer area (the bed is 235x235)
2. your probe can go max to the right side


original bltouch bracket have to set proble to nozzle offset around x -44 and y -7 (you can set in advance setting)
